stress-example/app.py

```diff
@@ -0,0 +1,57 @@
+__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import os
+import sys
+
+from jina.flow import Flow
+
+num_docs = int(os.environ.get('JINA_MAX_DOCS', 50000))
+image_src = 'data/**/*.png'
+
+
+def config():
+    parallel = 1 if sys.argv[1] == 'index' else 1
+    shards = 1
+
+    os.environ['JINA_PARALLEL'] = str(parallel)
+    os.environ['JINA_SHARDS'] = str(shards)
+    os.environ['WORKDIR'] = './workspace'
+    os.makedirs(os.environ['WORKDIR'], exist_ok=True)
+    os.environ['JINA_PORT'] = os.environ.get('JINA_PORT', str(45678))
+    os.environ['ENCODER'] = os.environ.get('ENCODER', 'jinaai/hub.executors.encoders.image.torchvision-mobilenet_v2')
```

Comment:
this encoder does not exist anymore, I think using directly `ImageTorchEncoder` should be okey